### PR TITLE
#118 Improve readme

### LINKED ISSUE #118: Improve readme

### PR DESCRIPTION
**Pull request by lecourtoisflo**
closes #118

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,35 +56,72 @@ export DYNAWO_CXX11_ENABLED=YES
...
+### Build
+
+launch the following command:
+
+``` bash
 $> ./myEnvDynawoAlgorithms.sh build
 ```
 
-For more details on the fic_MULTIPLE.xml and aggregatedResults.xml formats please refer to the documentation.
+All commands described in the rest of this README are accessible throught this script. To access all options of the script myEnvDFL.sh, type:
```
Comment (gautierbureau) [inline README.md:71]: myEnvDFL -> myEnvDynawoAlgorithms

File: README.md [hunk 2]
```diff
@@ -32,35 +56,72 @@ export DYNAWO_CXX11_ENABLED=YES
...
 ```
 
 ## Launching a systematic analysis
 
 ``` bash
 $> ./myEnvDynawoAlgorithms.sh SA --directory PATH_TO_SA_FOLDER --input fic_MULTIPLE.xml --output aggregatedResults.xml --nbThreads NB_THREADS_TO_USE
-
 ```
 
+fic_MULTIPLE.xml can be replaced by a archive file containing all the required input files
```
Comment (gautierbureau) [inline README.md:113]: an archive

File: README.md [hunk 3]
```diff
@@ -32,35 +56,72 @@ export DYNAWO_CXX11_ENABLED=YES
...
+
 ## Launching a margin calculation
+
 ``` bash
 $> ./myEnvDynawoAlgorithms.sh MC --directory PATH_TO_MC_FOLDER --input fic_MULTIPLE.xml --output aggregatedResults.xml --nbThreads NB_THREADS_TO_USE
-
 ```
 
+fic_MULTIPLE.xml can be replaced by a archive file containing all the required input files
```
Comment (gautierbureau) [inline README.md:121]: an archive

**Review by gautierbureau — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lecourtoisflo) [reply README.md:71]: OK

File: README.md (hunk 2 above)
Comment (lecourtoisflo) [reply README.md:113]: OK

File: README.md (hunk 3 above)
Comment (lecourtoisflo) [reply README.md:121]: OK

**Review by gautierbureau — APPROVED** [review]

**Pull request merged**